component-classes/index.js

```diff
@@ -387,7 +387,7 @@ export const modal = {
   transitionTitleCenter: 'justify-self-center',
   transitionTitleColSpan: 'col-span-2',
   title:
-    '-mt-4 sm:-mt-8 h-40 sm:h-48 grid gap-8 sm:gap-16 grid-cols-[auto_1fr_auto] items-center px-16 sm:px-32 border-b sm:border-b-0 shrink-0',
+    'py-8 sm:py-0 -mt-4 sm:-mt-8 min-h-40 sm:min-h-48 grid gap-8 sm:gap-16 grid-cols-[auto_1fr_auto] items-center px-16 sm:px-32 border-b sm:border-b-0 shrink-0',
```

Comment:
If you replace `items-center` with `items-start` here and add `self-center` to `transitionTitleCenter` key in line 387, then long title and buttons should align nicely:
<img width="200" alt="Screenshot 2024-01-03 at 12 06 05" src="https://github.com/warp-ds/css/assets/41303231/d893d094-a52a-4740-839e-ab43b5d53764">
<img width="200" alt="Screenshot 2024-01-03 at 12 06 05" src="https://github.com/warp-ds/css/assets/41303231/5df292fa-99a1-4487-a354-ced54d257e7a">

Reply:
Fixed in commit: [f2043db](https://github.com/warp-ds/css/pull/117/commits/f2043db84b704ed00fc0c01089a02d0c22036046) :)